src/components/select/select.js

```diff
@@ -1136,11 +1142,11 @@ function SelectProvider($$interimElementProvider) {
            */
           function mouseOnScrollbar() {
             var clickOnScrollbar = false;
-            if (ev && (ev.currentTarget.children.length > 0)) {
+            if (mouseDownXPosition != null && ev && (ev.currentTarget.children.length > 0)) {
```

Comment:
Hello ThomasBurleson,
I ve not made the check with another container Element. It was much much easier.
I skipped all the mouse position calculations and added a one-liner.
The mouse event already provided the originating element, so this was perfekt to check whether an event belongs to scrolling or not. I don't know why I ve not seen that before.